Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
Which version of Cargo did you use to compile the project?

Reply:
```
cainome git:(main) cargo --version
cargo 1.85.1 (d73d2caf9 2024-12-31)
```

Reply:
my bad, I don't use rustup and install rust/cargo from sources via nix so rust-toolchain.toml was ignored. Downgrading cargo, should work now
That actually has shown there's a deprecated action used here https://github.com/actions-rs/toolchain, so you'd never get recent (in the last 2 years) tool chains. I've created PR to fix this https://github.com/cartridge-gg/cainome/pull/86

Reply:
Thank you for the feedback and sorry for the delay @vaporif.

I've fixed the issue on the linting, since `0.14` removed the `V1` execution, the legacy codegen was actually hardcoded to use `V1`.

Reply:
I've updated the MSRV because of an issue publishing cainome release, which required an update also.